crates/matrix-sdk/src/encryption/identities/users.rs

```diff
@@ -507,7 +507,7 @@ impl OtherUserIdentity {
             }
             room.clone()
         } else {
-            self.client.create_dm(self.inner.user_id()).await?
+            self.client.create_dm(self.inner.user_id(), Encrypted::Yes).await?
```

Comment:
It seems to be quite an impactful change, does it?

cc @BillCarsonFr @andybalaam @jplatte

Reply:
Doesn't impact verification. But I don't think this is yet used by EX